schematools/contrib/django/models.py

```diff
@@ -53,6 +53,7 @@ def fetch_srid(dataset: DatasetSchema, field: DatasetFieldSchema) -> Dict[str, A
     "number": (models.FloatField, None),
     "boolean": (models.BooleanField, None),
     "array": (ArrayField, None),
+    "nmrelation": (models.ManyToManyField, None),
```

Comment:
Misschien beter `/relation/multiple` of iets anders meer duidelijk?

Reply:
Ik zie dat deze er helemaal uit kan, omdat ik het uiteindelijk toch anders heb aangepakt, net als bij de foreign keys. Ik haal deze weg, dan geef-ie ook geen onduidelijkheid meer :-)